py3dtilers/Common/README.md

```diff
@@ -0,0 +1,144 @@
+## [object_to_tile](Common/object_to_tile.py)
+### 🟠ObjectToTile
```

Comment:
Small color icons (nice eye candy for distinguishing ObectToTile from ObjectSToTile) don't get loaded on FireFox or Chrome.
Deux possible solutions:
  1. use html to have text in color (refer [here](https://stackoverflow.com/questions/35465557/how-to-apply-color-in-markdown))
  2. use github makrdown emoticons (e.g. like `:small_blue_diamond:` 🔹 or 🔺  (refer [here](https://github.com/markdown-templates/markdown-emojis)) also I'm not sure that gitlab supports them.
 
Note that storing small bullet point images in the repo to be displayed as makrdown pictures might prove difficult for fix for a correct layout.

Reply:
As of today, the difficulty remains.